libs/api-mocks/msw/handlers.ts

```diff
@@ -630,8 +630,8 @@ export const handlers = makeHandlers({
     return {
       nextPage,
       items: silos.map((silo) => ({
-        allocated: { cpus: 0, storage: 0, memory: 0 },
-        provisioned: { cpus: 0, storage: 0, memory: 0 },
+        allocated: { cpus: 52, storage: 26, memory: 130 },
+        provisioned: { cpus: 20, storage: 18, memory: 30 },
```

Comment:
Like @zephraph said, storage and memory are going to need to be in bytes I think. `130 * GiB`. Then you use `bytesToGiB` in the code.